types/types.go

```diff
@@ -646,7 +648,8 @@ type DiskSpecWithName struct {
 
 // UnmarshalToDisks input format should be:
 // `[{"path":"/mnt/disk1","allowScheduling":false},
-//   {"path":"/mnt/disk2","allowScheduling":false,"storageReserved":1024,"tags":["ssd","fast"]}]`
+//
+//	{"path":"/mnt/disk2","allowScheduling":false,"storageReserved":1024,"tags":["ssd","fast"]}]`
```

Comment:
Hmm?

Reply:
Reorganize the file to avoid make confused.